pkg/server/server.go

```diff
@@ -121,7 +121,8 @@ func (host *yamlLanguageHost) Run(ctx context.Context, req *pulumirpc.RunRequest
 		}
 	}
 
-	template, diags, err := host.loadTemplate()
+	compilerEnv := []string{fmt.Sprintf(`PULUMI_STACK=%s`, req.GetStack())}
```

Comment:
There's other values we know at compile time like the org name, project, config, etc. Should we be setting envvars for all of these? Should we just have one big envvar with a JSON like object for all of them?

Reply:
Added org, project and config (as JSON) to the environment